src/components/Button/Button.tsx

```diff
@@ -1,11 +1,12 @@
+import { ReactNode } from 'react';
 import styles from './button.module.scss';
 
 const classNames = require('classnames');
```

Comment:
Overlooked this. Never do this. We are not mixing ES6 modules and CommonJS.
This is like a sandwich where one side is peanut butter jelly and the other is Mettwurst.